core/src/parcel.rs

```diff
@@ -78,7 +78,7 @@ pub fn parcel_error_message(error: &ParcelError) -> String {
         AlreadyImported => "Already imported".into(),
         Old => "No longer valid".into(),
         TooCheapToReplace => "Gas price too low to replace".into(),
-        InvalidNetworkId => "Parcel of this network ID is not allowed on this chain.".into(),
+        InvalidNetworkId => "Network ID is not allowed on this chain.".into(),
```

Comment:
This network ID is not allowed on this chain

Reply:
@kseo Fixed it